src/Orc.NuGetExplorer/Native/CredentialsPrompter.cs

```diff
@@ -1,10 +1,4 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="CredentialsPrompter.cs" company="WildGums">
-//   Copyright (c) 2008 - 2015 WildGums. All rights reserved.
-// </copyright>
-// --------------------------------------------------------------------------------------------------------------------
-
-namespace Orc.NuGetExplorer.Native
+﻿namespace Orc.NuGetExplorer.Native
```

Comment:
Rename to Win32 (similar to other projects)

Reply:
Some of classes inside these folder should be removed, but this one probably should be moved to Windows folder. As it's used win32 api inside, but it's not really a interop class